presets/index.d.ts

```diff
@@ -1,7 +1,7 @@
-import type { TsJestPresets } from 'ts-jest/dist/types';
+import type { Config } from '@jest/types';
 
 declare const _default: {
-  defaults: TsJestPresets;
-  defaultsESM: TsJestPresets;
+  defaults: Config.InitialOptions;
```

Comment:
This one is better https://github.com/kulshekhar/ts-jest/blob/c7ba4593ffdcc2bc81c1193863ab2941c3376976/src/types.ts#L211